nilearn/plotting/displays.py

```diff
@@ -1216,6 +1221,10 @@ def add_graph(self, adjacency_matrix, node_coords,
                               vmin=edge_vmin, vmax=edge_vmax,
                               **edge_kwargs)
 
+        # At the moment threshold=0, we fix this based upon discussion
```

Comment:
If you are unsure about something, it is better to add a comment on github to get the discussion going rather than in the source file.

Why not use threshold=edge_threshold here?